js/main.js

```diff
@@ -68,6 +76,22 @@ jQuery(document).ready(function ($) {
         $('.main-nav a span:first').removeClass('dd-active');
     });
 
+	$('.drop-nav').on('click', function () {
+		if (isMobile == true) {
+			console.log("DropNav clicked.");
+			
+			
+			
+		}
+    });
+	function getsupportedprop(proparray){

```

Comment:
Add comment to document what the reason for this function is (I need to be better at this myself).